jenkins-pipelines/longevity-100gb-4h-fips.jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!groovy
+
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+// This scenario can only be triggered with a scylla repo, not an ami
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-100gb-4h.yaml", "configurations/longevity-fips-and-encryptions.yaml", "configurations/local-ear.yaml"]'''
```

Comment:
first time i see 3 config files!!! hope it works flawlessly

Reply:
Exactly what I thought! I am making history

Reply:
Why should it work ? 

I'm quite sure we had a few of those in siren-tests

Reply:
was just an observation...